packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -91,7 +91,10 @@ export const defaultChatItemMessageContainer = (overlapAvatarAndMessage: boolean
     marginRight: '0rem',
     marginLeft: `${messageAvatarGap}rem`,
     width: `calc(100% - ${AVATAR_WIDTH_REM + MESSAGE_AMOUNT_OUT_FROM_EDGE_REM + messageAvatarGap}rem)`,
-    zIndex: CHAT_MESSAGE_ZINDEX
+    zIndex: CHAT_MESSAGE_ZINDEX,
+    '@media (forced-colors: active)': {
+      forcedColorAdjust: 'none'
```

Comment:
does stopping forced colours maintain the required colour ratio for the different modes? @edwardlee-msft I believe its like 3:1 or something for items like this? or was it 5:1?

Reply:
How can we check that?
Teams app also shows the same colors in high contrast modes.

Reply:
I use the accessibility insights tool for windows, you can download it here: https://accessibilityinsights.io/docs/windows/overview/

Reply:
Updated the description with screenshots from the tool. LMKWYT

Reply:
For me Teams has the same bug where messages have no border:
![image](https://user-images.githubusercontent.com/2684369/180322207-301607e7-fa0c-4af4-98f0-6b6a8d277b7d.png)

Reply:
Correct.